content/blog/wordpress-comparison.mdx

```diff
@@ -0,0 +1,38 @@
+2. Headless CMS for Flexibility: TinaCMS gives you full control over how your content is displayed. This headless architecture allows you to integrate your CMS with any frontend framework, whether it's React, Vue, or Next.js. For those moving from WordPress or a managed WordPress host like WP Engine, this gives you to better performance, scalability, and frontend customization.
+3. A Better Developer Experience: While WordPress relies on PHP, TinaCMS leverages modern JavaScript frameworks like React. This provides a more intuitive development experience for teams that prefer working within the JavaScript ecosystem. If you’re developing on platforms like WP Engine but want to shift to a more JavaScript-focused stack, TinaCMS offers a modern approach without the complexities often associated with traditional CMS platforms.
+4. Simplified Content Management: WordPress offers a vast array of plugins and features, but this can sometimes lead to site bloat and maintenance overhead. TinaCMS, on the other hand, provides a streamlined, lightweight solution that keeps your setup clean and focused. This is particularly attractive for developers looking to avoid the complexity of managing plugins and customizations on platforms like WP Engine. TinaCMS also uses Markdown, which is easy to use and great for quick formatting and documentation.
+5. Freedom from Platform Lock-in: Unlike WordPress, which is often tied to specific hosting solutions like WP Engine, TinaCMS is fully hosting-agnostic. Its Git-backed architecture ensures that your content and code remain portable, giving you the freedom to choose the best infrastructure for your project without being locked into a particular platform.
+
+#### Is TinaCMS Right for You?
+
+If you're a developer or part of a team currently using WordPress or WP Engine, TinaCMS offers a modern, flexible alternative that can streamline your workflow. By integrating content management directly into your Git workflow, TinaCMS provides the control and flexibility you need to build fast, scalable websites—without the overhead and complexities of traditional CMS platforms.
+
+***
+
+#### How to Migrate from WordPress to TinaCMS
+
+Transitioning from WordPress to TinaCMS doesn’t have to be complicated. With TinaCMS’s headless and Git-first approach, migrating your content and code can be done in a few steps. To learn more about the migration process, check out our step-by-step guide on how to migrate from WordPress to TinaCMS.
```

Comment:
Feels like this should like somewhere...
For now we could comment it out and get this blog live without it

Reply:
Afaik that guide doesn't exist?

Reply:
Does this exist now?